frontend/apps/filemanager/filemanager.lua

```diff
@@ -455,6 +461,10 @@ function FileManager:init()
     FileManager.instance = self
 end
 
+function FileManager:onToggleWifi()
+    return self.networklistener:onToggleWifi()
```

Comment:
This feels a bit risky, using the same name for the event handler.
Dispatcher/Gestures do send a ToggleWifi event - before this PR, networklistener is the only catcher. With this, depending on stuff, we don't really know which of FileManager or networklistener will get it, and possible side effects.
So, I think it's best to use another name `FileManager:onKeyToggleWifi()` or something. Same in Reader*.
(I like @hius07 don't really like these added event handlers, but it looks like there is no other way.)